docs/pages/application-access/guides/api-access.mdx

```diff
@@ -8,6 +8,10 @@ description: How to access REST APIs with Teleport Application Access.
 Teleport Application Access can be used to access applications' (REST or Teleport's own gRPC) APIs with
 tools like [curl](https://man7.org/linux/man-pages/man1/curl.1.html) or Postman.
 
+<Details scopeOnly={true} scope={["cloud"]} title="Teleport Cloud Compatibility">
```

Comment:
Is the idea that we're going to put a note like this on every guide that is compatible with cloud?

1-2 years from now I will expect that 90% or more of our guides work on cloud, won't this get repetitive? Would it be easier to assume cloud compatibility and only call out cases that don't work on cloud?